src/Avalonia.Controls/TopLevel.cs

```diff
@@ -116,7 +116,8 @@ public abstract class TopLevel : ContentControl,
         private TargetWeakEventSubscriber<TopLevel, ResourcesChangedEventArgs>? _resourcesChangesSubscriber;
         private IStorageProvider? _storageProvider;
         private LayoutDiagnosticBridge? _layoutDiagnosticBridge;
-        
+        private static WeakReference<TopLevel>? _appRoot;
```

Comment:
What is "app root" and why the first toplevel is the root?

Reply:
Yes